UnicornRender/system/source/profiler/MonitorProfiler.cpp

```diff
@@ -21,14 +21,14 @@ namespace system
 MonitorProfiler::MonitorProfiler(Manager& manager)
     : m_systemManager( manager )
 {
-    LOG_INFO("MonitorProfiler created");
+    spdlog::get(utility::Logger::LoggerName())->info("MonitorProfiler created");
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.